deployment/k8s-config/harbor/values-keel-dev.yaml

```diff
@@ -78,7 +143,13 @@ persistence:
 #***************************************************
 core:
   priorityClassName: uber-user-preempt-high
```

Comment:
I think this priority class name needs to be added to each component of harbor, not just core.

Reply:
Updated each harbor component with both affinity and priority class. The affinity of cadc-harbor-jobservice is not set correctly. Will look into it.